apps/docs/src/components/Showcase.astro

```diff
@@ -0,0 +1,28 @@
+---
+import { SIDEBAR } from '../config';
+import { getLanguageFromURL } from '../languages';
+import ShowcaseCard from './ShowcaseCard.astro';
+
+const currentPage = Astro.url.pathname;
+const langCode = getLanguageFromURL(currentPage);
+const components = SIDEBAR[langCode]?.Components;
+---
+
+<div
+  class="preview grid grid-cols-1 gap-x-1 gap-y-2 sm:grid-cols-2 xl:grid-cols-3"
+>
+  {
+    components.map(({ text, sublinks, link }) => {
+      if (link) return <ShowcaseCard title={text} href={link} />;
+
+      if (sublinks)
+        return (
+          <>
+            {sublinks?.map(({ text, link }) => (
```

Comment:
since you already checked for sublinks the `?` shouldn't be necessary right?

Reply:
Right, good catch :)